src/bundle/Controller/DashboardController.php

```diff
@@ -9,14 +9,36 @@
 namespace EzSystems\HybridPlatformUiBundle\Controller;
 
 use EzSystems\HybridPlatformUi\Components\MainContent;
+use EzSystems\HybridPlatformUi\Dashboard\Dashboard;
 use EzSystems\PlatformUIBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
 
 class DashboardController extends Controller
 {
+    protected $dashboard;
+
+    public function __construct(Dashboard $dashboard)
+    {
+        $this->dashboard = $dashboard;
+    }
+
     public function viewDashboardAction(MainContent $mainContent)
     {
         $mainContent->setTemplate('EzSystemsHybridPlatformUiBundle::dashboard.html.twig');
+        $mainContent->setParameters([ 'dashboard' => $this->dashboard ]);
 
         return $mainContent;
     }
+
+    public function viewTabAction(Request $request, $section, $tab)
```

Comment:
Argument names should probably be made more clear as `$sectionIdentifier, $tabIdentifier`, maybe also type hint on them